osu.Game.Rulesets.Osu/Skinning/Default/KiaiFlash.cs

```diff
@@ -30,13 +35,19 @@ public KiaiFlash()
             };
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            flashOpacity = config.GetBindable<float>(OsuSetting.GlowStrength);
```

Comment:
Should not be reassigning to the value. Use `config.BindWith` instead (or don't initialise the value until here).

Reply:
Was unaware of this method to bind 👍